ivy/functional/frontends/numpy/ndarray/ndarray.py

```diff
@@ -448,3 +448,6 @@ def __setitem__(self, key, value):
     
     def __mod__(self, value, /):
         return np_frontend.mod(self._ivy_array, value, out=self)
+
+    def round(self, decimals=0):
+        return np_frontend.round(self._ivy_array,decimals=decimals)
```

Comment:
There isn't a function known as round() in the numpy frontend implemented yet which makes it a dummy call here :slightly_smiling_face:

Reply:
Am I supposed to implement that function? Sorry I am new to this.